src/Illuminate/Support/Str.php

```diff
@@ -245,7 +245,7 @@ public static function random($length = 16)
      */
     public static function randomBytes($length = 16)
     {
-        if (PHP_MAJOR_VERSION >= 7) {
+        if (PHP_MAJOR_VERSION >= 7 || defined('RANDOM_COMPAT_READ_BUFFER')) {
```

Comment:
Could you add `paragonie/random_compat` as a suggested dependency in the support composer.json please?